azure-toolkit-libs/azure-toolkit-compute-lib/src/main/java/com/microsoft/azure/toolkit/lib/compute/virtualmachine/VirtualMachine.java

```diff
@@ -100,6 +101,7 @@ public boolean isSshEnabled() {
     }
 
     @Nullable
+    @Cacheable(cacheName = "vm/{}/hostIp", key = "${this.getName()}")
```

Comment:
use resource id instead of name as cache key?

Reply:
fixed